src/interfaces/todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
```

Comment:
```suggestion
  userId: number,
  todoUser?: User,
```
and use this type everwhere